misc/shlib/shlib.bash

```diff
@@ -211,3 +200,11 @@ arch_go() {
 red() {
     echo -ne "\e[31m$*\e[0m"
 }
+
+# in_ci
+#
+# Returns 1 if in CI and 0 otherwise
+in_ci() {
+    [ -z "${BUILDKITE-}" ] && return 1
+    return 0
```

Comment:
Not worth making this change now, but just as an FYI pretty sure this function reduces to:

```suggestion
[[ "${BUILDKITE-}" ]]
```